src/index.ts

```diff
@@ -209,6 +210,18 @@ const setupTsRunner = (
     }
   }
 
+  if (process.versions.pnp) {
+    try {
+      const resolvedPnp = require.resolve('pnpapi')
+      if (
+        !NODE_OPTIONS?.includes(resolvedPnp) &&
+        !execArgv.includes(resolvedPnp)
+      ) {
+        execArgv = [...execArgv, `-r ${resolvedPnp}`]
+      }
+    } catch {}
```

Comment:
If pnp can't be loaded, fail gracefully -- it's possible someone is using a non-yarn pnp implementation

Reply:
Maybe we can include [`debug`](https://github.com/debug-js/debug) support?

Reply:
I found no reason to ignore such error, let's keep looking forward.